test/integration/hostgroup_test.rb

```diff
@@ -1,4 +1,4 @@
-require 'test_helper'
+require 'integration_test_helper'
 
 class HostgroupIntegrationTest < ActionDispatch::IntegrationTest
```

Comment:
This class also has a section of JS-enabled tests, could you either move them or define a second class in this file for it?